tests/python/pants_test/backend/python/tasks/test_pytest_run.py

```diff
@@ -650,6 +653,67 @@ def test_one(self):
     self.assertEqual([1, 2], all_statements)
     self.assertEqual([], not_run_statements)
 
+  def test_coverage_omit_test_sources(self):
```

Comment:
Adding a non-synthetic `__init__.py` that does something meaningful - maybe exports a constant used by math.py - would be good to prove we do the right thing in a srcs+tests package like this. The right thing being including the `__init__.py` in coverage reporting I think.

Reply:
That would definitely be the right thing. Will add.

Reply:
But I expect that to work because the lib target owns the __init__.py.